cube/bmp_8.cpp

```diff
@@ -77,14 +73,14 @@ unsigned char **Bmp8::CreateBmpBuffer(unsigned char init_val) {
 
   buff = (unsigned char **) new unsigned char *[hgt_ * sizeof(*buff)];
   if (!buff) {
```

Comment:
The codebase doesn't have exceptions handling (Goole have a C++ policy that forbid their use).